html/leave_group_example.html

```diff
@@ -1,5 +1,5 @@
-<div>
-    <form>
+<div style="color: white">
```

Comment:
If this is an example of something that works in group_edit it should be deleted before merging. Otherwise I don't understand what its purpose is

Reply:
okay